app/services/edition_unpublisher.rb

```diff
@@ -29,11 +27,6 @@ def fire_transition!
     mark_attachments_as_present_at_unpublish
   end
 
-  def prepare_edition
-    edition.force_published = false
-    edition.reset_version_numbers
-  end
```

Comment:
My [brief search of the code](https://github.com/search?q=repo%3Aalphagov%2Fwhitehall+%28published_major_version+OR+published_minor_version%29&type=code) suggests the version numbers on editions are used to decide whether the "First published at" field is shown to users. And also maybe when building the "change notes" history that get sent to the Publishing API. But I don't think the values themselves are sent downstream to the Publishing API.

In any case, I'm wondering what the desired behaviour should be here? 🤔

It's worth noting that the `reset_version_numbers` method doesn't actually set them back to `0`. Instead it [rolls them back](https://github.com/alphagov/whitehall/blob/fe3e2d61dec9f4c881d9d16c15258496bb2d3284/app/models/edition/publishing.rb#L87-L90) to whatever versions the previous edition had.

Reply:
This whole thing is a bit mind boggling, but my instinct is to say we leave it in rather than remove it.

But 🤯 🤷🏻

Reply:
I'm not sure I agree with you on this one, though it's not a strong disagreement. The unpublished version remains a genuine version that did exist and was publicly available, to my mind. If we reset the version number, we'll end up with the previous edition and the unpublished edition having the same version number, which doesn't seem right and might lead to confusing change notes?

Reply:
Reverted in latest push